compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -727,7 +734,6 @@ pub fn codegen_crate<B: ExtraBackendMethods>(
         ongoing_codegen.check_for_errors(tcx.sess);
 
         let cgu_reuse = cgu_reuse[i];
-        tcx.sess.cgu_reuse_tracker.set_actual_reuse(cgu.name().as_str(), cgu_reuse);
 
         match cgu_reuse {
```

Comment:
Why is `PreLto` reachable in this match? `cgu_reuse` comes from `determine_cgu_reuse` too, doesn't it?